.vscode/settings.json

```diff
@@ -0,0 +1,44 @@
+{

```

Comment:
I see that you managed to remove one of the files in the .vscode folder in one of your recent commits. Not the end of the world but if you could also remove this one that would be good - wasn't v clear in my earlier comments I meant both